config.js

```diff
@@ -62,6 +62,11 @@ const ConfigOptions = () => {
         },
 
         /**
+         * Copy takes an object of assets to copy from src to dist folders
+         */
+        copy: {},
```

Comment:
Have you tested that this works correctly without adding a line to the `update` function lower down in this module?